### PR TITLE
feat: allow overriding api reference slug in docs

### PR DESCRIPTION
**Pull request by abvthecity**
FER-1659

### CODE REVIEW

**Comment by dsinghvi** [comment]
Once we add some docs for this (https://buildwithfern.com/learn/docs/api-references/generate-api-ref), let's merge

File: fern/pages/fern-docs/content/generate-api-ref.mdx [hunk 1]
```diff
@@ -9,9 +9,21 @@ and errors automatically populated from your [OpenAPI specification](/learn/over
...
 
-<CodeBlock title='docs.yml'>
-```yml
+```yml docs.yml
 navigation:
   - api: API Reference
 ```
-</CodeBlock>
+
+| Property         | Description                                                                                   |
```
Comment (abvthecity) [inline fern/pages/fern-docs/content/generate-api-ref.mdx:17]: Added some placeholder docs here. Examples needed.

Comment (abvthecity) [reply fern/pages/fern-docs/content/generate-api-ref.mdx:17]: Screenshot 2024-05-09 at 1 45 02 PM

**Review by dsinghvi — APPROVED** [review]
the table looks great, thank you!

**Pull request merged**